components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -27,7 +64,13 @@ service Policies {
     option (chef.automate.api.policy).action = "create";
     option (chef.automate.api.iam.policy).resource = "iam:policies";
     option (chef.automate.api.iam.policy).action = "iam:policies:create";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  Get a policy
```

Comment:
We still need to give at least one sentence beyond just the heading--even if it adds no real value--because the docs look broken without it.
<img width="689" alt="image" src="https://user-images.githubusercontent.com/6817500/73149964-b3803b00-4078-11ea-9992-ca3ce5acd3b7.png">

Reply:
🤔

Reply:
I agree that this looks odd, but some of our API calls don't require any parameters at all (The list calls, and they look really weird), and the ID in fetching seems self-explanatory.

I agree we should be consistent tho 🤷‍♂

Reply:
I'm gonna copy-paste `Get a policy` twice then. There's not much else to say here.